src/mapping/cellranger_multi/script.py

```diff
@@ -197,7 +208,31 @@ def resolve_input_directories_to_fastq_paths(input_paths: list[str]) -> list[Pat
 
     return input_paths
 
-def process_params(par: dict[str, Any]) -> str:
+def make_paths_absolute(par: dict[str, Any], config: Path | str):
+    with open(config, 'r', encoding="utf-8") as open_viash_config:
+        config = yaml.safe_load(open_viash_config)
+
+    arguments = {
+        arg["name"].removeprefix("-").removeprefix("-"): arg
+        for group in config["functionality"]["argument_groups"]
+        for arg in group["arguments"]
+    }
+    file_arguments = {arg_name: arg for arg_name, arg in arguments.items()
+                      if arg['type'] == 'file'}
+    for file_arg_name, arg_options in file_arguments.items():
```

Comment:
No need to really create a separate `file_arguments` for this.

```suggestion
    for file_arg_name, arg_options in arguments.items():
      if arg_options["type"] != "file":
        continue
```

When done like this, I'd be inclined to rename `file_arg_name, arg_options` to `arg_name, arg`.